.github/workflows/release.yml

```
@@ -0,0 +1,80 @@
+name: release
+
+on:
+  push:
+    tags:
+      - '**'
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+  publish-artifacts:
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: 19
+
+      - name: Setup Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
+
+      - name: Upload Artifacts
+        run: |
+          ./gradlew publish --stacktrace --warning-mode all --no-parallel --no-daemon
```

```suggestion
          ./gradlew publish --stacktrace --warning-mode all --no-daemon
```
The limitation requiring sequential execution during publish has been removed.